CHANGELOG.md

```diff
@@ -47,7 +81,7 @@
 * Address test dependency issue #4556 [#4606](https://github.com/near/nearcore/pull/4606). [#4622](https://github.com/near/nearcore/pull/4622).
 * Fix neard shutdown issue [#4429](https://github.com/near/nearcore/pull/4429). #[4442](https://github.com/near/nearcore/pull/4442)
 
-## `1.20.0` [07-26-2021]
```

Comment:
what's wrong with the highlighted version numbers? (as in the backticks)

Reply:
No reason to format it as code.